app/code/Magento/Paypal/Block/Customer/Account/BillingAgreementsLinkInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Paypal\Block\Customer\Account;
+
+use Magento\Customer\Block\Account\SortLinkInterface;
+
+/**
+ * BillingAgreementsLinkInterface
+ * @api
+ */
+interface BillingAgreementsLinkInterface extends SortLinkInterface
```

Comment:
I'm not sure it's a good idea to introduce a new interface just for storing config paths. Maybe we should move it to a separate config model or to the block class itself. 

@sidolov could you share your thoughts on this matter, please? 
Thank you

Reply:
@rogyar totally agree with you. If constants used in one model only, we should define them on the particular model as private.